### ISSUE TITLE
Advanced search parameters field: you can remove items like "*/.git" but you cannot add a string like "*/.git"

### ISSUE THREAD

**Issue by Wikinaut** [pixelb/fslint#107]
In the "Advanced search parameters" field, i could easily remove items like ```"*/.git"``` but I cannot re-add an entry like ```"*/.git"```.

#### bug
Currently, the directory selection window is presented, but for re-adding the ASP I suggest to have a free text input field.

**Comment by pixelb** [comment]
On my gnome system I'm presented with a file dialog, but can enter free form text.
I notice that I can toggle this free form text field with Ctrl-L.
I'm not sure if it's displayed by default, though I've not noticed it missing ever.
I'll look to see if I can configure this field to be visible by default

**Comment by Wikinaut** [comment]
@pixelb you wrote
> On my gnome system I'm presented with a file dialog, but can enter free form text.
Okay, on my Opensuse Tumbleweed (German localisation) the text suggests to "add a new path" - so perhaps before I got paranoid because I did not want to add a path like "*/.git" -- this seems to be only matter of a better text.

> I notice that I can toggle this free form text field with Ctrl-L.
Uh, yes, great! it was not opened by default, but helped. So a good solution, in my view, is to i) change the text from "add a new path" to "add a new path to the advanced search parameters" and ii) add a text "type Cntl-L" to open a free text entry field.

I close this issue because my "problem" is solved. Pls reopen, if you as the author want to have it open.
 
Did I already say, that the new version (from git) is working very well? I will keep on posting some reports, and will also try to submit pull requests.
